source/Modules/Devon4Net.Infrastructure.AWS.CDK/Resources/Management/ECS/ECSHandlerManager.cs

```diff
@@ -19,7 +19,7 @@ internal CapacityProviderStrategy CreateCapacityProviderStrategy(AsgCapacityProv
             return HandlerResources.AwsCdkEcsHandler.CreateCapacityProviderStrategy(capacityProvider, capWeigth, capBase);
         }
 
-        public IService AddElasticContainerEc2Service(string id, string serviceName, ICluster cluster, TaskDefinition taskDefinition, int? healthCheckGracePeriod, List<CapacityProviderStrategy> capacityProviderStrategies, int? desiredCount, bool useDistinctInstances, List<string> placementStrategies) //NOSONAR number of params
+        public IService AddElasticContainerEc2Service(string id, string serviceName, ICluster cluster, TaskDefinition taskDefinition, int? healthCheckGracePeriod, List<CapacityProviderStrategy> capacityProviderStrategies, int? desiredCount, bool useDistinctInstances, List<string> placementStrategies)
```

Comment:
Please put the sonar comment to avoid the restriction of maximum number of params